website/content/en/scripts/step11-create-provisioner.sh

```diff
@@ -0,0 +1,26 @@
+cat <<EOF | kubectl apply -f -
+apiVersion: karpenter.sh/v1alpha5
+kind: Provisioner
+metadata:
+  name: default
+spec:
+  requirements:
+    - key: karpenter.k8s.aws/instance-category
+      operator: In
+      values: [c, m, r]
+    - key: karpenter.k8s.aws/instance-generation
+      operator: Gt
+      values: ["2"]
+  providerRef:
```

Comment:
the main guide says that after scaling down with step14-deprovisioning says that emptiness will kick in but it's not here. 
```suggestion
  ttlSecondsAfterEmpty: 30
  providerRef:
```

Reply:
Okay. I added that line.